gemsfx/src/main/java/com/dlsc/gemsfx/PDFView.java

```diff
@@ -284,6 +284,35 @@ public final boolean gotoLastPage() {
         return currentPage != getPage();
     }
 
+    /**
+     * Convenience method to decrease the zoom factor for value specified as {@code delta}.
+     *
+     * @param delta zoom factor (decrese) delta
+     * @return true if the operation actually did cause a zoom change
+     */
+
+    public final boolean decreaseZoomFactor(double delta) {
```

Comment:
These two methods are utility methods that need to go into the skin. JavaFX controls usually never expose doThis(), doThat() convenience methods on the controls themselves but only provide properties.

Reply:
Ack.